### DISCUSSION TITLE
Feature Request - Add icons to File Browser panesl

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#3876] [Ideas]
![image](https://github.com/Bforartists/Bforartists/assets/35357409/30297e0a-543c-45b4-8309-d65103e3e84e)

Could we add icons here? The order of these panels changed, so it's hard to find the right panel. Would be good to have a representative icon for the panels Bookmakrs, System, Volumens and Recent.

**Comment by ReinerBforartists** [comment]
Really? :D
It's four panels. I would see this as visual noise. And when you start it here then you have to continue elsewhere. Tough one. 

We could revert the order. This is determined in the order of the classes afaik. But i find the current order not bad.

**Reply by Draise14** [reply]
Order is ok.
